libs/designer/src/lib/ui/panel/nodeDetailsPanel/tabs/parametersTab/index.tsx

```diff
@@ -454,6 +463,27 @@ export const getEditorAndOptions = (
       },
     };
   }
+  // add a flag for Manual triggers v/s Hybrid triggers to determine the schema object to be added for Serialization/Deserialization
+  if (equals(editor, 'floatingactionmenu')) {
+    let isManualTrigger = false;
+    if (
+      !(
+        equals(operationType?.toLowerCase(), constants.NODE.TYPE.REQUEST) &&
+        equals(operationKind?.toLowerCase(), constants.NODE.KIND.APICONNECTION)
+      )
+    ) {
+      isManualTrigger = true;
+    }
+    return {
+      editor: editor,
+      editorOptions: {
+        ...editorOptions,
+        ...{
+          isManualTrigger,
```

Comment:
Why not create a new option under `x-ms-editor-options` for the manual trigger manifest so we don't need this custom logic.

Reply:
Thank you for pointing out this option, I added the new `x-ms-editor-options' of `'isRequestApiConnectionTrigger` for the hybridTriggers instead and removed the customization here. For manual triggers it can remain undefined